capabilities.js

```diff
@@ -4,6 +4,19 @@ define('capabilities', ['settings'], function(settings) {
         return !!m && m.matches;
     }
 
+    function detectOS() {
```

Comment:
It seems like something that might be updated later. For this use case it appears as though it detects the OS version.